saleor/account/forms.py

```diff
@@ -127,3 +127,27 @@ def send_mail(
         # template, we remove it from the context.
         del context['user']
         emails.send_password_reset_email.delay(context, to_email)
+
+
+class EmailChangeForm(forms.Form):
+    """ Allow changing email """
+
+    email = forms.EmailField(
+        label=pgettext('New email', 'Email'), max_length=75)
+
+    error_messages = {
+        'email': pgettext_lazy('New email address cannot be the same'
```

Comment:
`pgettext` takes two arguments, first is the context of the translated string, second is the string to be translated. You're missing the first one here